cucumber/authenticators_gcp/features/step_definitions/authn_gcp_steps.rb

```diff
@@ -24,19 +24,83 @@
   set_annotation_to_resource("authn-gcp/#{annotation_name}", annotation_value)
 end
 
-# Sets all GCP annotations
+Given(/^I set "authn-gcp\/(service-account-id|service-account-email|project-id|instance-name)" annotation to function host "([^"]*)"$/) do |annotation_name, hostname|
```

Comment:
why is this step duplicated? how is it different from the step above besides the word `function`?